components/02-molecules/pager/pager.yml

```diff
@@ -0,0 +1,9 @@
+current: 1
+items:
+  previous: true
+  next: true
```

Comment:
I don't think we want this on the first example, because it only makes sense to show the arrows if there's "something to navigate to". If you're on the first page, there's nothing to navigate back to, for example.

Reply:
I updated the examples in the commit I pushed up.

Reply:
Looks great to me!